src/pretix/base/models/items.py

```diff
@@ -2294,3 +2294,30 @@ class ItemVariationMetaValue(LoggedModel):
 
     class Meta:
         unique_together = ('variation', 'property')
+
+
+class ItemProgramTime(models.Model):
```

Comment:
This requires addition to the code paths that are executed when copying events and copying items so the information is carried over.